src/commands/prepare.ts

```diff
@@ -126,9 +126,10 @@ async function createReleaseBranch(
   }
   if (branchHead) {
     let errorMsg = `Branch already exists: ${branchName}. `;
+    const remoteName = process.env.CRAFT_REMOTE || 'origin';
```

Comment:
Could we perhaps move this get-or-default logic to a separate function? Otherwise it's duplicated across different files.

Reply:
Within the same file ok, updated; among the two files I didn't see a clear way forward, suggestions welcome.